docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -40,68 +40,68 @@ configuration file will result in checkpoints being sorted by the loss metric in
      smaller_is_better: true
 
 The following snippet of Python code can be run after the specified experiment has generated a
-checkpoint. It returns an instance of :class:`~determined.experimental.client.Checkpoint`
-representing the checkpoint that has the best validation metric.
+checkpoint. It gets a list of sorted :class:`~determined.experimental.client.Checkpoint` instances
+associated with the experiment and selects the one with the best validation metric.
```

Comment:
After generating a checkpoint for the specified experiment, you can run the Python code below. This code retrieves a list of sorted :class:`~determined.experimental.client.Checkpoint` instances associated with the experiment and selects the checkpoint with the best validation metric.